boards/arm/mimxrt1062_fmurt6/mimxrt1062_fmurt6-pinctrl.dtsi

```diff
@@ -0,0 +1,522 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2023 NXP
+ *
+ */
+
+#include <nxp/nxp_imx/rt/mimxrt1062dvl6a-pinctrl.dtsi>
+
+/* Flash RESET pin is DNP here unlike RT1050 */
+
+&pinctrl {
+	/* ADC1 inputs 0 and 15 */
+	pinmux_adc1: pinmux_adc1 {
+		group0 {
+			pinmux = <&iomuxc_gpio_ad_b1_11_adc1_in0>;
+			bias-disable;
+			drive-strength = "r0-6";
+			slew-rate = "slow";
+			nxp,speed = "100-mhz";
+		};
+	};
+
+	pinmux_enet: pinmux_enet {
+		group0 {
+			pinmux = <&iomuxc_gpio_b0_15_enet2_ref_clk2>;
+			drive-strength = "r0-6";
+			slew-rate = "fast";
+			nxp,speed = "50-mhz";
+			bias-pull-down-value = "100k";
+			input-enable;
+		};
+		group1 {
+			pinmux = <&iomuxc_gpio_b1_01_enet2_rx_data0>,
+				<&iomuxc_gpio_b1_02_enet2_rx_data1>,
+				<&iomuxc_gpio_b1_03_enet2_rx_en>,
+				<&iomuxc_gpio_b0_12_enet2_tx_data0>,
+				<&iomuxc_gpio_b0_13_enet2_tx_data1>,
+				<&iomuxc_gpio_b0_14_enet2_tx_en>,
+				<&iomuxc_gpio_b0_00_enet2_mdc>;
+			drive-strength = "r0-5";
+			//bias-pull-up;
+			bias-pull-up-value = "100k";
+			slew-rate = "fast";
+			nxp,speed = "200-mhz";
+		};
+		group2 {
+			pinmux = <&iomuxc_gpio_b1_00_enet2_rx_er>;
+			bias-pull-down-value = "100k";
+			drive-strength = "r0-5";
+			slew-rate = "fast";
+			nxp,speed = "200-mhz";
+		};
+		group3 {
+			pinmux = <&iomuxc_gpio_b0_01_enet2_mdio>;
+			drive-strength = "r0-5";
+			drive-open-drain;
+			bias-pull-up-value = "100k";
+			slew-rate = "fast";
+			nxp,speed = "50-mhz";
+		};
+	};
+
+	pinmux_sensor: pinmux_sensor {
+		group0	{
+			pinmux = <&iomuxc_gpio_emc_13_gpio4_io13>,
+				<&iomuxc_gpio_emc_09_gpio4_io09>,
+				<&iomuxc_gpio_emc_06_gpio4_io06>,
+				<&iomuxc_gpio_emc_41_gpio3_io27>,
+				<&iomuxc_gpio_ad_b0_00_gpio1_io00>,
+				<&iomuxc_snvs_pmic_on_req_gpio5_io01>;
+			//bias-disable;
```

Comment:
Please remove this commented line if it is not needed

Reply:
Removed